components/PostPreview.tsx

```diff
@@ -16,23 +16,28 @@ const PostPreview: React.FC<Props> = ({ post }) => {
 
   return (
     <div className="shadow-lg border-[0.5px] rounded-xl relative bg-white flex flex-col justify-start items-start md:ml-10">
-      {new Date().getTime() - new Date(post.publishedAt).getTime() <
+      {new Date().getTime() - new Date(post.createdAt).getTime() <
         1000 * 60 * 60 * 24 * 7 && (
         <div className="w-12 h-12 bg-sky-400 rounded-full flex justify-center items-center text-white right-4 -top-4 absolute shadow-md uppercase text-sm font-medium rotate-12">
-          New
+          {language === 'en' ? 'New' : 'Nuevo'}
         </div>
       )}
       <div className="absolute w-20 h-20 rounded-full shadow-md bg-white md:-left-10 md:top-6 -top-10 left-4 flex flex-col items-center justify-center border-[0.5px]">
         <h3 className="uppercase text-gray-400 text-sm font-light">
           {(language === `en` ? englishMonths : spanishMonths)[
-            new Date(post.publishedAt).getMonth()
+            new Date(post.createdAt).getMonth()
           ].substring(0, 3)}
         </h3>
-        <h3 className="text-2xl text-sky-500">{new Date(post.publishedAt).getDate()}</h3>
+        <h3 className="text-2xl text-sky-500">{new Date(post.createdAt).getDate()}</h3>
       </div>
       <div className="p-6 md:pl-16 pt-16 md:pt-6 pb-4">
-        <div>
+        <div className="flex justify-start items-center space-x-4">
           <h2 className="text-xl font-inter text-gray-900">{post[language].title}</h2>
+          {post.category === 'teaching' && (
+            <span className="py-0.5 px-4 text-sm border border-sky-300 bg-sky-50 rounded-full text-sky-500 font-medium">
```

Comment:
is this exactly the same as the file above? maybe worth extracting out a little component?